models/address.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import "github.com/wopta/goworkspace/lib"
+
+type Address struct {
+	StreetName   string `json:"streetName" firestore:"streetName" bigquery:"-"`
+	StreetNumber string `json:"streetNumber" firestore:"streetNumber" bigquery:"-"`
+	City         string `json:"city" firestore:"city" bigquery:"-"`
+	PostalCode   string `json:"postalCode" firestore:"postalCode" bigquery:"-"`
+	Locality     string `json:"locality" firestore:"locality" bigquery:"-"`
+	CityCode     string `json:"cityCode" firestore:"cityCode" bigquery:"-"`
+	Area         string `json:"area" firestore:"area" bigquery:"-"`
+}
+
+func (a *Address) Sanitize() {
+	a.StreetName = lib.ToUpper(a.StreetName)
+	a.StreetNumber = lib.ToUpper(a.StreetNumber)
+	a.City = lib.ToUpper(a.City)
+	a.PostalCode = lib.TrimSpace(a.PostalCode)
```

Comment:
since passing a number string (ex.: "20210") to ToUpper does not throw an error, we could use the same function here since some postal codes (ex.: UK) use letters in conjunction with numbers.